packages/pwa-kit-mcp/README.md

```diff
@@ -1,206 +1,130 @@
 
-    - "Create a new PWA Kit app."
-    - "Start a new storefront using a preset."
-    - "What templates are available for PWA Kit?"
-    - "What PWA-Kit presets are available?"
-    - "Create a PWA-Kit project using the `retail-react-app-demo` preset in the `~/test-project` directory."
-- `run_site_test`: Run site performance or accessibility test for a given site URL (e.g. [https://pwa-kit.mobify-storefront.com](https://pwa-kit.mobify-storefront.com))
--`create_sample_storefront_page`: Helps developers create a new sample PWA Storefront page. This feature guides develoopers through a few simple questions and then generates code for the page based on the name of the page, components included, and route.
+* **`create_sample_storefront_page`**:
+  Interactive tool to generate a new PWA storefront page with custom routing and components.
 
-## Setup
+* **`development_guidelines`**:
+  Provides best practices and guidance for building PWA Kit storefronts.
 
-Install dependencies and build under `pwa-kit` root directory:
+* **`run_site_test`**:
+  Runs performance and accessibility audits on a provided site URL.
+  *Example: `https://pwa-kit.mobify-storefront.com`*
 
-```bash
-cd {{pwa-kit root directory}}
-npm ci
-```
-
-## Run the MCP Server
-
-### Method 1: Run MCP Server from Cursor
-
-1. Open the Cursor application.
 
-2. In the Cursor Menu on the top menu bar, click **Settings** > **Cursor Settings...**. 
+## ▶️ Running the MCP Server
 
-<img src="https://raw.githubusercontent.com/SalesforceCommerceCloud/pwa-kit/refs/heads/develop/packages/pwa-kit-mcp/docs/images/cursor-settings.png" alt="Cursor Settings Screenshot" width="50%" />
+### 🖥️ From Cursor IDE
 
-3. Click **Tools & Integrations** > **MCP Tools** > **New MCP Server**.
+1. Open **Cursor**.
 
-<img src="https://raw.githubusercontent.com/SalesforceCommerceCloud/pwa-kit/refs/heads/develop/packages/pwa-kit-mcp/docs/images/cursor-mcp-tools.png" alt="Cursor MCP Tools Screenshot" width="50%" />
+2. Navigate to **Settings > Cursor Settings...**
+![](https://raw.githubusercontent.com/SalesforceCommerceCloud/pwa-kit/refs/heads/develop/packages/pwa-kit-mcp/docs/images/cursor-settings.png)
 
-The `mcp.json` file opens. Add this definition to your `mcp.json` file and replace {{parent-dir-to-mcp}} and {{path-to-app-directory}} placeholders with correct values.
+3. Go to **Tools & Integrations > MCP Tools > New MCP Server**
+![](https://raw.githubusercontent.com/SalesforceCommerceCloud/pwa-kit/refs/heads/develop/packages/pwa-kit-mcp/docs/images/cursor-mcp-tools.png)
 
+4. Update your `mcp.json` like this (edit the placeholders as needed):
 ```json
 {
   "mcpServers": {
-    "pwa-kit-mcp": {
+    "pwa-kit": {
       "command": "npx",
       "args": ["-y", "@salesforce/pwa-kit-mcp"],
       "env": {
         "PWA_STOREFRONT_APP_PATH": "{{path-to-app-directory}}"
       }
     }
   }
-} 
+}
```

Comment:
We should have an example of what the app directory is (is it the storefront folder or the app subfolder). For example, we can add something similar to:

An example of the `{{path-to-app-directory}}` placeholder value is `"/Users/username/mcp-server-folder/mystorefront/app"`.

Reply:
@bendvc I see the text is now different than my example text. I pasted it below.

{{path-to-app-directory}} should be replaced with the absolute path to your generated project. For example: "/Users/username/mcp-server-folder/mystorefront/app".

I think we should say the `app` subfolder. 

So I suggest the following:

Replace {{path-to-app-directory}} with the absolute path to your generated project's `app` subfolder. For example: "/Users/username/mcp-server-folder/mystorefront/app".